docs/examples/index.md

```diff
@@ -15,8 +16,29 @@ Check out these examples of different editor integrations. See the predefined bu
 </info-box>
 </span>
 
+## Predefined builds
+
+{@link installation/getting-started/predefined-builds Predefined CKEditor&nbsp;5 builds} are a set of ready-to-use rich text editors. Every “build” provides a single type of editor with a set of features and a default configuration. They provide convenient solutions that can be installed with no effort and that satisfy the most common editing use cases. See all of them in action: {@link examples/builds/classic-editor classic}, {@link examples/builds/inline-editor inline}, {@link examples/builds/balloon-editor balloon}, {@link examples/builds/balloon-block-editor balloon block}, {@link examples/builds/document-editor document}, and {@link examples/builds/multi-root-editor multi-root} editors! Learn their differences by using our examples and analyzing the source code of the builds.
+
+## Advanced custom builds
+
+CKEditor&nbsp;5 is a configurable framework that was created with collaboration in mind. It lets you build a custom editor of any type, with a wide set of features and the toolbar type that you need where multiple authors can easily work on the same rich text documents. Check out our {@link examples/builds/full-featured-editor feature-rich editor} that sports as many plugins as it is possible to put together. Want to invite some friends? Try the {@link examples/builds/collaborative-document-editor collaborative document editor} that brings the experience of the most popular online editing solutions to the table. And witness the flexibility of the UI in the {@link examples/custom/bottom-toolbar-editor toolbar-oriented example}.
+
+## Framework examples
+
+{@link framework/index CKEditor&nbsp;5 Framework} is a highly flexible and universal platform that provides a set of components allowing you to create any kind of rich text editor. It enables the integrators to build different, custom-tailored editing solutions with {@link examples/custom-ui custom UI} or {@link examples/theme-customization a theme} that suit their specific needs. It also provides tools for the creation and integration of {@link examples/framework/data-from-external-source some user-made}, {@link examples/framework/content-placeholder custom features} and for customizing {@link examples/chat-with-mentions existing ones}. See how CKEditor 5 can integrate a {@link examples/framework/using-react-in-a-widget React library} to enrich both the contents and the editing experience.
+
+## Experimental solutions
+
+If you need even more, try our experimental plugins, like the integration of the {@link examples/experiments/mermaid Mermaid library} that can be used to display flowcharts and diagrams.
+
+## How-tos
+
+There come times when you don't need deep-dive guides, elaborate solutions, and complicated examples. Sometimes you just need to know how to set the height of CKEditor&nbsp;5? Or, maybe want to learn how to focus the editor? This is what the {@link examples/how-tos How-to} section is for!
```

Comment:
> Sometimes you just need to know how to set the height of CKEditor 5? Or, maybe want to learn how to focus the editor? This is what the How-to section is for!

Missing comma + the sentence sounds incomplete: "Sometimes, you just need to know how to set the height of CKEditor 5. Or, maybe you want to learn how to focus the editor? This is what the How-to section is for!"